pom.xml

```diff
@@ -70,7 +70,7 @@ Note: The maven build currently needs to be started from the javaslang root dir
         <java.version>1.8</java.version>
         <jmh.version>1.13</jmh.version>
         <junit.version>4.12</junit.version>
-        <gwt.version>2.8.0-rc1</gwt.version>
+        <gwt.version>2.8.0-535f2fa9</gwt.version>
```

Comment:
Maybe better rely on the 2.8.0-SNAPSHOT for now, pending 2.8.0 final release that will include that change.